helper/pluginutils/catalog/register_nvidia_linux.go

```diff
@@ -0,0 +1,14 @@
+// +build !nonvidia
```

Comment:
nonvidia?

Reply:
yeah - as in `no nvidia`; not sure what the build tags casing convention is but we can go with `no_nvidia` or `no-nvidia` to make it clearer?